bookwyrm/templates/snippets/shelve_button/shelve_button_options.html

```diff
@@ -2,8 +2,8 @@
 {% load i18n %}
 {% for shelf in shelves %}
 {% comparison_bool shelf.identifier active_shelf.shelf.identifier as is_current %}
-{% if dropdown %}<li role="menuitem">{% endif %}
-    <div class="{% if dropdown %}dropdown-item pt-0 pb-0{% elif active_shelf.shelf.identifier|next_shelf != shelf.identifier %}is-hidden{% endif %}">
+{% if dropdown %}<li role="menuitem" class="dropdown-item p-0">{% endif %}
+    <div class="{% if active_shelf.shelf.identifier|next_shelf != shelf.identifier %}is-hidden{% endif %}">
```

Comment:
This if/else logic isn't quite right. The way you've changed it is much more readable, I think you just need to add `if not dropdown` to the second if statement, as in:
`"{% if not dropdown and active_shelf.shelf.identifier|next_shelf != shelf.identifier %}`